modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ManagementToolbar.js

```diff
@@ -73,27 +73,29 @@ function ManagementToolbar({
 		<>
 			<ClayManagementToolbar active={active}>
 				<ClayManagementToolbar.ItemList>
-					<SelectionControls
-						actionDropdownItems={actionDropdownItems}
-						active={active}
-						clearSelectionURL={clearSelectionURL}
-						disabled={disabled}
-						initialCheckboxStatus={initialCheckboxStatus}
-						initialSelectAllButtonVisible={
-							initialSelectAllButtonVisible
-						}
-						initialSelectedItems={initialSelectedItems}
-						itemsTotal={itemsTotal}
-						onCheckboxChange={onCheckboxChange}
-						onClearButtonClick={onClearSelectionButtonClick}
-						onSelectAllButtonClick={onSelectAllButtonClick}
-						searchContainerId={searchContainerId}
-						selectAllURL={selectAllURL}
-						selectable={selectable}
-						setActionDropdownItems={setActionDropdownItems}
-						setActive={setActive}
-						supportsBulkActions={supportsBulkActions}
-					/>
+					{selectable && (
```

Comment:
With this check here, we no longer need to pass `selectable` prop to `<SelectionControls>` and can remove [the `{selectable && (...)}` check in component](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/SelectionControls.js#L128).

Reply:
@markocikos indeed. Thanks for the review.

Reply:
Updated in [aea295f](https://github.com/liferay-frontend/liferay-portal/pull/797/commits/aea295f62e0a463a37ef902a9402458e63d8abf3)